libs/ha/ha_comp_libs.py

```diff
@@ -152,16 +160,24 @@ def get_ha_log_prop(node_obj, log_name: str, kvalue: int, fault_tolerance: bool=
                 LOGGER.info("ha pvc: %s", ha_pvc)
                 break
         if fault_tolerance:
-            # For one pod operation there will 9 lines of log
+            # For one pod operation there will be 9 lines of log
             kvalue *= 9
         if health_monitor:
-            # For one pod operation there will 4 lines of log
-            kvalue *= 4
+            # For one pod operation there will be 4 lines of log
+            if kubectl_delete:
+                kvalue*=8
```

Comment:
Please add comment as added for earlier kvalue

Reply:
Done